--- lat_controller.cc	2021-03-26 06:59:36.359233865 +0000
+++ lat_controller_buggy.cc	2021-03-26 07:12:12.012813163 +0000
@@ -414,8 +414,10 @@
     */
     cf_ = -control_conf_->lat_controller_conf().cf();
     cr_ = -control_conf_->lat_controller_conf().cr();
-    matrix_a_(0, 1) = 0.0;
-    matrix_a_coeff_(0, 2) = 1.0;
+    
+    AWARN << "Triggered bug d36f3661e958e13e3f647c910257c8db32fb9788 (a)";
+    //matrix_a_(0, 1) = 0.0;
+    //matrix_a_coeff_(0, 2) = 1.0;
   } else {
     /*
     A matrix (Gear Drive)
@@ -717,7 +719,8 @@
   } else {
     v = std::max(injector_->vehicle_state()->linear_velocity(),
                  minimum_speed_protection_);
-    matrix_a_(0, 2) = 0.0;
+    AWARN << "Triggered bug d36f3661e958e13e3f647c910257c8db32fb9788 (b)";
+    //  matrix_a_(0, 2) = 0.0;
   }
   matrix_a_(1, 1) = matrix_a_coeff_(1, 1) / v;
   matrix_a_(1, 3) = matrix_a_coeff_(1, 3) / v;
